edxval/api.py

```diff
@@ -182,6 +143,47 @@ def update_video_status(edx_video_id, status):
     video.save()
 
 
+def get_course_video_image_url(course_id, edx_video_id):
+    """
+    Returns course video image url or None if no image found
+    """
+    try:
+        video_image = CourseVideo.objects.select_related('video_image').get(
+            course_id=course_id, video__edx_video_id=edx_video_id
+        ).video_image
+        return video_image.image_url()
+    except ObjectDoesNotExist:
+        return None
+
+
+def update_video_image(edx_video_id, course_id, image_data, file_name):
```

Comment:
We need to specify in the doc string about why do we need filename because an obvious thought that comes into mind is; this filename would be used as it is for the image but it is only needed to decide the image's extension.